docs/conf.py

```diff
@@ -30,19 +30,14 @@
 from docutils.utils import get_source_line
 
 
-def _warn_node(self, msg, node):
-    """Do not warn on external images."""
-    if not msg.startswith('nonlocal image URI found:'):
-        self._warnfunc(msg, '{0}:{1}'.format(get_source_line(node)))
-
-sphinx.environment.BuildEnvironment.warn_node = _warn_node
-
-
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
 
+# Do not warn on external images.
```

Comment:
If we pin it we will never fix it. Good example is OAuthlib in OAuthClient and OAuthServer ...

> ... avoid conflicts is by doing a `pip freeze` ...

better `pip-compile` & `pip-sync` ;)